integration-tests/__tests__/coverage_transform_instrumented.test.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const {readFileSync} = require('fs');
+const path = require('path');
+const skipOnWindows = require('../../scripts/SkipOnWindows');
+const {cleanup, run} = require('../utils');
+const runJest = require('../runJest');
+
+const dir = path.resolve(__dirname, '../coverage-transform-instrumented');
+const coverageDir = path.join(dir, 'coverage');
+
+skipOnWindows.suite();
```

Comment:
Do we need to skip it on windows?

Reply:
Not sure. Do we have reasons documented on why we would want to skip on windows?

I just did it here, since I copied from another coverage integration test.

If someone could run it on windows and verify that would be great.

Reply:
AppVeyor CI runs our test suite on Windows :). It's better to remove all the `skipOnWindows` references unless something breaks on AppVeyor for some weird reasons and you don't have time to fix it properly